assets/js/blocks/single-product/index.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { BLOCK_ICON } from './constants';
+import metadata from './block.json';
+import edit from './edit';
+import save from './save';
+
+// @ts-expect-error: `registerBlockType` is a function that is typed in WordPress core.
```

Comment:
I know this comment is based on code prior to this PR, but to me it's not clear why there is a TS error here. `registerBlockType` is a function that is typed in WordPress core, but why do we have to add `@ts-expect-error`? Sorry if it's something obvious that I'm missing. :sweat_smile:

Reply:
Yeah... no need to fix it in this PR, but it would be good to follow-up and improve it. :slightly_smiling_face:

Reply:
The function `registerBlockType` isn't typed well. So, we should improve it upstream (not sure it is easy). The comment is useful because if it is fixed, a TS error will be shown and we can remove the comment. (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments)